main/webapp/modules/core/styles/index/open-project-ui.less

```diff
@@ -111,113 +111,168 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 }
 
 #metadata-body table {
-  border-collapse:collapse
+  border-collapse: collapse;
+  table-layout: fixed;
 }
 
-#metadata-body td {
-    padding: 1px
+table.list-table{
+    margin:0;
 }
 
-#metadata-body tr:nth-child(even) {background: #f2f2f2}
-#metadata-body tr:nth-child(odd) {background: #FFF}
+#metadata-body td {
+  padding: 1px;
+}
 
-#metadata-body table {
-  table-layout: fixed;
+#metadata-body tr:nth-child(even) {
+  background: #f2f2f2;
+}
+#metadata-body tr:nth-child(odd) {
+  background: #fff;
 }
 
 #metadata-body table > tbody > tr > td {
   word-wrap: break-word;
 }
 
 #metadata-body table > tbody > tr:nth-child(1) > th:nth-child(2) {
- width: 80%
- }
+  width: 80%;
+}
 
 #metadata-body table > tbody > tr:nth-child(1) > th:nth-child(3) {
- width: 4%
- }
-
+  width: 4%;
+}
 
 /* Making metadata table scrollable*/
-.scrollable-table{
-	max-height: 70vh;
-	overflow-y: auto;
-	overflow-x: hidden;
+.scrollable-table {
+  max-height: 70vh;
+  overflow-y: auto;
+  overflow-x: hidden;
 }
 
 /* Making metadata table header sticky and darker*/
-#metadata-body table > tbody > tr >th{
-	position: -webkit-sticky;
-	position:sticky;
-	top:0;
-	color:black;
-	background:#b7b7b7;
+#metadata-body table > tbody > tr > th {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  color: black;
+  background: #b7b7b7;
 }
 
- 
- /* Project Tags */
+/* Project Tags */
 #projectTags {
-	position: relative;
-	padding: 3px 10px;
-	margin: 5px;
-	font-size: 130%;
-	border-radius: 5px;
-	-moz-border-radius: 5px;
-	-webkit-border-radius: 5px;
-	background: #bcf;
-	min-height: 18px;
-	height: auto;
-	overflow:auto;
+  position: relative;
+  padding: 3px 10px;
+  margin: 0px;
+  font-size: 130%;
+  border-radius: 0px;
+  -moz-border-radius: 0px;
+  -webkit-border-radius: 0px;
```

Comment:
Same here: it is not clear to me why those changes are needed to introduce a search function.

Reply:
I've table out the `border-radius` styles but perhaps the `font-size` could stay

without `font-size` styles
![image](https://user-images.githubusercontent.com/66228179/162852036-d2ef2855-9ef8-400a-9e15-b80574b76dfa.png)

with `font-size`
![image](https://user-images.githubusercontent.com/66228179/162852149-8bc42a0c-1735-479e-bfe0-9769fc1d00a1.png)